lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_publication
+short_description: Add or remove PostgreSQL publication from a remote host
+description:
+   - Add or remove PostgreSQL publication from a remote host.
+version_added: "2.5"
+options:
+  name:
+    description:
+      - Name of the publication to add or remove.
+    required: true
+    default: null
```

Comment:
Both `required: true` and `default: null` are not needed anymore. You can leave them if you like, but I thought I'd mention it.